sklearn/utils/validation.py

```diff
@@ -305,8 +318,9 @@ def check_X_y(X, y, accept_sparse=None, dtype=None, order=None, copy=False,
         If the input is sparse but not in the allowed format, it will be
         converted to the first listed format.
 
-    dtype : string, type or None (default=none)
+    dtype : string, type or None (default="numeric")
         Data type of result. If None, the dtype of the input is preserved.
```

Comment:
I would be more specific and say "Data type of the checked input data X" instead of "Data type of result".